frontend/scss/organisms/_o-custom-tours-landing.scss

```diff
@@ -164,4 +171,161 @@
   }
 }
 
+// -- Custom tours: Custom footer
+
+.m-cta-banner--aic-ct {
+  @include breakpoint('medium-') {
+    min-height: 240px;
+    height: fit-content;
+  }
+
+  @include breakpoint('small-') {
```

Comment:
I would generally use a mobile-first approach. i.e. only use greater than breakpoints and treat the smallest screen as the default.